satpy/readers/li_base_nc.py

```diff
@@ -371,6 +371,9 @@ def inverse_projection(self, azimuth, elevation, proj_dict):
         azimuth = azimuth.values * point_height
         elevation = elevation.values * point_height
 
+        # In the MTG world, azimuth is defined as positive towards west, while proj expects it positive towards east
+        azimuth *= -1
```

Comment:
I was going to comment that in-place multiplication with Dask arrays is dangerous, but it seems the array is already computed (`azimuth.values` is accessed). Is that necessary for some further computations?

Reply:
That `self.inverse_projection` method is called inside a `da.map_blocks`: https://github.com/pytroll/satpy/blob/d29268cd98a175a686898e10ae31123aa837189e/satpy/readers/li_base_nc.py#L339-L344
so believe it should remain daskified. IIRC, we needed to get the `.values` inside there as the following `Proj` call accepts only computed values.

Reply:
I would have thought `da.map_blocks()` would pass Numpy arrays to the method being used :thinking:  Haven't used it in a while so might remember incorrectly. And if it works, then it must be correct because Numpy arrays wouldn't have `.values` attribute causing a crash :sweat_smile: